packages/@aws-cdk/aws-appmesh/lib/http-route-match-method.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Supported values for matching routes based on the HTTP request method
+ */
+export enum HttpRouteMatchMethod {
```

Comment:
The name of this enum is quite weird... the `*Match` suffix seems to suggest to me this is a union-like class, with static factory methods named like `startingWith()`, etc. But it's not, it's just simple values.

Given that, how about renaming it to just `HttpRouteMethod` instead? Same as we have `HttpRouteProtocol`.

Reply:
I like this idea!